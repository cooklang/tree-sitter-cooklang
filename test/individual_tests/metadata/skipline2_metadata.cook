>> hello: test

recipe line
recipe line

>> test: hi
