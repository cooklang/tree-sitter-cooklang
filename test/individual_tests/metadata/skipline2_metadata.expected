(recipe (metadata) (step) (step) (metadata))